config/interpolate_funcs.go

```diff
@@ -23,6 +23,7 @@ func init() {
 		"element": interpolationFuncElement(),
 		"replace": interpolationFuncReplace(),
 		"split":   interpolationFuncSplit(),
+		"toint":   interpolationFuncToint(),
```

Comment:
If you like the explicit conversion (and don't think Terraform should be doing this implicitly) then I also like to_int more - I will rename and add tests some time in the next couple of days. :)